### PR TITLE
Apply the same filter and buttons style to Contacts as in Accounts

### PR DESCRIPTION
**Pull request by ekowalsk**
image

### CODE REVIEW

File: packages/react-components/src/FilterInput.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+interface Props {
+  className?: string;
+  filterOn: string;
+  label: string;
+  setFilter: (filter: string) => void;
+}
+
+function Filter ({ className = '', filterOn, label, setFilter }: Props) {
+  return (
+    <div className={`${className} filter--tags`}>
```
Comment (MiZiet) [inline packages/react-components/src/FilterInput.tsx:18]:
```suggestion
    <div className={className}>
```
do we use `filter--tags` anywhere?

File: packages/react-components/src/FilterInput.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    :not(:only-child) {
+      margin-left: 1.5rem;
+    }
+
+    .ui--Input {
+      margin: 0;
+      height: 3.893rem;
+    }
+  }
+`);
```
Comment (MiZiet) [inline packages/react-components/src/FilterInput.tsx:43]:
```suggestion
export default React.memo(styled(Filter)`
  width: 29.5rem;

  &:not(:only-child) {
    margin-left: 1.5rem;
  }

  .ui--Input {
    margin: 0;
    height: 3.893rem;
  }
`);

```

Comment (ekowalsk) [reply packages/react-components/src/FilterInput.tsx]: ✅

**Review by MiZiet — APPROVED** [review]
Nice

**Review by jacogr — APPROVED** [review]
Nice. Apart from aligning, it removes real uglies in the code. Brilliant.

**Pull request merged**